maven/com/mucommander/sevenzipjbinding/sevenzipjbinding-all-platforms/16.02-2.01/sevenzipjbinding-all-platforms-16.02-2.01.pom

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.mucommander.sevenzipjbinding</groupId>
```

Comment:
I didn't do it eventually but at some point when splitting the mono-repo into bundles, I thought to publish the format and protocol bundles in maven-central and back then the groupId was supposed to be `com.mucommander` - I think it would be better to set the groupId here also to `com.mucommander` so that if we ever publish the other artifacts, they'll all have the same groupId